src/hooks/viewport/useDisableBodyScroll.tsx

```diff
@@ -4,7 +4,7 @@ import { ATTENDEE_LAYOUT_CLASSNAME } from "settings";
 
 // @debt: we might need to find a solution that doesn't use body element/classname, but modifies
 // only the #root element
-export const useDisableBodyScroll = (isOpen: boolean) => {
+export const useDisableBodyScroll = (isOpen: boolean = true) => {
```

Comment:
[NOTE]
More like note to self for some future refactor. But also FYI:

the function API is the semi-famous boolean trap that basically favors
`useDisableBodyScroll({ isOpen: true} )` over `useDisableBodyScroll(true)`
because it something like self documenting code

Reply:
[288812e](https://github.com/sparkletown/sparkle/pull/2966/commits/288812e57154d1d6485340f66d5ac0198a607253)